cnf-tests/testsuites/pkg/features/features.go

```diff
@@ -0,0 +1,222 @@
+package features
+
+import (
+	"fmt"
+
+	sriovClean "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/clean"
```

Comment:
nit: please split the imports to

1. golang packages
2. external projects
3. internal